pkgs/applications/misc/zathura/core/default.nix

```diff
@@ -95,6 +99,16 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional stdenv.hostPlatform.isLinux libseccomp
     ++ lib.optional stdenv.hostPlatform.isDarwin gtk-mac-integration;
 
+  # add support for more image formats
+  GDK_PIXBUF_MODULE_FILE = gnome._gdkPixbufCacheBuilder_DO_NOT_USE {
```

Comment:
Given the usage of gnome internals here cc @jtojnar

Reply:
I moved it to `env`. Is this something that is recommended in general?

Reply:
With structured attrs non-`env` things are only bash variables and not env variables anymore.
I failed to find good documentation on it unfortunately, the best explanation I found after a quick search is the following post on my former employer's blog: https://nixos.mayflower.consulting/blog/2020/01/20/structured-attrs/